subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/ui/SubscriptionWebViewViewModel.kt

```diff
@@ -121,6 +123,12 @@ class SubscriptionWebViewViewModel @Inject constructor(
                 PIR -> GoToPIR
                 else -> null
             }
+            when (commandToSend) {
+                GoToITR -> pixelSender.reportOnboardingIdtrClick()
```

Comment:
Should all this have `is` ?

Reply:
`is` is redundant for a Kotlin `object`, so IDE omits it when generating branches for `when` expression.
```
        data object GoToITR : Command()
        data object GoToPIR : Command()
        data class GoToNetP(val activityParams: ActivityParams) : Command()
```